pubpub/pubpub
# feat: Pub title decorations

This PR adds **bold** and _italics_ decorations to Pub titles!

These decorations can be added through conventional keyboard shortcuts (Meta+B, Meta+I), or through copy/pasting of existing HTML. All HTML other than `b`, `i`, `strong`, and `em` will be stripped out of pasted content. Undo and redo of formatting commands is supported.

https://user-images.githubusercontent.com/6402908/169891770-62656def-48b7-4987-9f7a-0c547bbf8607.mov

_Test Plan_

1. Follow the test steps from #2040 
2. Ensure the title saves on blur. Attempting to navigate away from the draft page should trigger a warning prompt.
3. Ensure Meta+B and Meta+I will bold/italics selections within the Pub title editor.
4. Ensure the Pub title editor is read-only on the release view.
5. Ensure undo/redo work as expected.
6. Attempt to paste HTML content containing tags other than the supported decoration tags. Ensure those tags (including attributes) are stripped.
7. Confirm that Pub titles render with decorations in the Activity tab.
